docs/UserGuide.md

```diff
@@ -217,22 +222,22 @@ Example of the usage:
 ### Un-marking a marked item: `UNMARK`
 Un-marks a marked-as-bought item from the list at the specified index.
 After being marked as bought, the item will have the status `[B]`
-to indicate that it is marked as bought. After unmarking the marked-as-bought
-item, the status of the item becomes `[0]`.
+to indicate that it is marked as bought. After un-marking the marked-as-bought
+item, the status of the item becomes `[X]`.
 
 Format: `UNMARK INDEX`
 
-* The `INDEX` should be an number.
+* The `INDEX` should be a **positive whole number**.
 * The `INDEX` should not be out of bounds of the shopping list.
-  * Out of bounds indexes include negative indexes & indexes that are greater than the size of the shopping list.
-* Indexes that are not numbers or are out of bounds will produce an error message indicating the error of the index.
-* Unmarking an item whose status was previously `[0]` will be successful, assuring the success of the unmark command executed.
+  * Out of bounds Indices include negative Indices & Indices that are greater than the size of the shopping list.
+* Indices that are not numbers or are out of bounds will produce an error message indicating the error of the index.
+* Un-marking an item whose status was previously `[X]` will be successful, assuring the success of the un-mark command executed.
 
 Example of the usage:
 
 1. `UNMARK 3`
-   * This unmarks the 3rd item in your list as unbought.
-   * The status of the 3rd item is now `[0]`
+   * This un-marks the 3rd item in your list as unbought.
```

Comment:
Again, perhaps its better for this to be mark as not bought yet, instead of unmarks as unbought

Reply:
Ok noted, thanks!!